rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ProjectSpecTest.java

```diff
@@ -283,6 +289,35 @@ public void before() throws TException, IOException {
         licenseInfoFile.setGeneratedOutput(new byte[0]);
         given(this.licenseInfoMockService.getLicenseInfoFile(anyObject(), anyObject(), anyObject(), anyObject(),
                 anyObject(),anyObject())).willReturn(licenseInfoFile);
+
+        Source ownerSrc1 = new Source();
+        ownerSrc1.setFieldValue(3, "9988776655");
+        Source usedBySrc1 = new Source();
+        usedBySrc1.setFieldValue(1, project.getId());
+        LicenseInfoUsage licenseInfoUsage1 = new LicenseInfoUsage(new HashSet<String>());
+        licenseInfoUsage1.setProjectPath("11223344:22334455");
+        licenseInfoUsage1.setExcludedLicenseIds(Sets.newHashSet("22334455", "9988776655"));
+        UsageData usageData1 = new UsageData();
+        usageData1.setLicenseInfo(licenseInfoUsage1);
+        AttachmentUsage attachmentUsage1 = new AttachmentUsage(ownerSrc1, "aa1122334455bbcc", usedBySrc1);
```

Comment:
May be it can be written as `Source.releaseId("9988776655")` instead of creating a separate Source and setting the field value based on the thrift id.

Reply:
Changes made as suggested.

Reply:
👍